src/raftstore/store/worker/apply.rs

```diff
@@ -321,6 +348,7 @@ impl ApplyDelegate {
         if committed_entries.is_empty() {
             return vec![];
         }
+        apply_ctx.prepare_for(self);
```

Comment:
why prepare here?

Reply:
Add a new `ApplyCallback` struct for current region.